hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/regionserver/LogRollBackupSubprocedurePool.java

```diff
@@ -62,10 +62,9 @@ public LogRollBackupSubprocedurePool(String name, Configuration conf) {
           LogRollRegionServerProcedureManager.BACKUP_TIMEOUT_MILLIS_DEFAULT);
     int threads = conf.getInt(CONCURENT_BACKUP_TASKS_KEY, DEFAULT_CONCURRENT_BACKUP_TASKS);
     this.name = name;
-    executor =
-        new ThreadPoolExecutor(1, threads, keepAlive, TimeUnit.SECONDS,
-            new LinkedBlockingQueue<>(),
-            Threads.newDaemonThreadFactory("rs(" + name + ")-backup"));
+    executor = new ThreadPoolExecutor(1, threads, keepAlive, TimeUnit.SECONDS,
+      new LinkedBlockingQueue<>(),
+      new ThreadFactoryBuilder().setNameFormat("rs(" + name + ")-backup-pool-%d").build());
```

Comment:
Should the thread names follow existing format ?
People may have got used to the current format during debugging.

Reply:
The reason why I appended `-pool` is to actually adhere with current format only. We add `-pool` in Threads class internally.

Reply:
@tedyu could you please review?
Thanks

Reply:
The change is fine.